docs/administration/configuration/configuring-private-docker-registries/index.md

```diff
@@ -0,0 +1,49 @@
+# Configuring private docker registries
+
+REANA is able to execute workflows which make use of Docker images hosted on private Docker image registries.
+You will just need to save the necessary credentials in Kubernetes secrets, and then instruct REANA to pass these secrets to each job's pod.
+
+## Obtaining the credentials
+
+First of all, you will need to obtain the authentication credentials, and how to do so varies depending on which registry you are using.
+One possibility would be to create a special service account to access the registry.
+As an example, the [REANA account](https://gitlab.cern.ch/reana) is used to authenticate to the private registry provided by CERN's GitLab instance, as shown in [Private Docker registries](../../../advanced-usage/access-control/private-docker-registries/#cern-gitlab).
+
+## Configuring Kubernetes and REANA
+
+After obtaining the credentials, you will have to save them in a Kubernetes secret, as explained in the related Kubernetes documentation page [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+In particular, this is how you would create the secret from an already existing Docker configuration file, after running `docker login`:
+
+```{ .console .copy-to-clipboard }
+$ kubectl create secret generic <your-credentials-secret> \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+As an alternative, you can create the secret by providing the username and password needed to authenticate to the private registry:
+
+```{ .console .copy-to-clipboard }
+$ kubectl create secret docker-registry <your-credentials-secret> \
+    --docker-server=<your-registry-server> \
+    --docker-username=<your-name> \
+    --docker-password=<your-password> \
+    --docker-email=<your-email>
+```
```

Comment:
In [another discussion](https://github.com/reanahub/docs.reana.io/pull/165#discussion_r1226590136), we considered the idea of using the `!!! tip` admonition for `kubectl`-related tips and how-tos throughout the admin-related documentation. I think it could fit well here, do you agree?

Reply:
Personally I don't like the use of `<variable>` in shell command examples, because it may be confusing WRT shell redirections.

Better to use `john.doe@example.org` , `johndoe`  etc so that readers will see a "realistic" example.

(Ditto `<path/to/.docker/config.json>` can better be `$HOME/.docker/config.json` for example.)

Reply:
> In [another discussion](https://github.com/reanahub/docs.reana.io/pull/165#discussion_r1226590136), we considered the idea of using the `!!! tip` admonition for `kubectl`-related tips and how-tos throughout the admin-related documentation. I think it could fit well here, do you agree?

Given that this is a necessary step to reach the end of the guide and this isn't an "additional tip" as in the other case, I think it's better to keep these commands as part of the main text instead of using `!!! tip` boxes.